indico/modules/designer/client/js/index.js

```diff
@@ -33,6 +33,13 @@
 
   var DEFAULT_PIXEL_CM = 50;
 
+  // Should add "is_image" property in the json and the PlaceHolder
+  var imageTypes = ["ticket_qr_code","event_logo"];
```

Comment:
What about actually passing the list of image placeholders to this function as well instead of hardcoding it? I had a look at the surrounding code and it would be pretty straightforward (build the list of image placeholder names in python, and pass it just like we pass `placeholders`).

Reply:
Ahhhh!! Now what you wrote above IS CLEAR :)
YES!! Totally agree... ! This will be flexible and allow additional images declared in the plugin.
PERFECT. I'll take a look at it.